andela/ah-rambo-frontend
# #168315879 Implement Reset Password Page

#### What does this PR do?
- Implement reset password page
- Add unit tests

#### Description of Task proposed in this pull request?
- Add actions and reducers
- Implement the reset password page
- Add unit tests for actions, reducers and components

#### How should this be manually tested (Quality Assurance)?
- Run `npm run start:dev'
- A browser tab will launch, then navigate to this route ```/request-password``` to request for a password reset link
- Input a valid email address and confirm the email address is a registered one, then submit.
- If successful, check your mail to get the link.
- Copy the token on the url sent and add to this route- ```/reset-password?token=< token here>```, open the route and change to a new password


#### What are the relevant pivotal tracker stories?
- [#168315879](https://www.pivotaltracker.com/story/show/168315879)

#### Any background context you want to add (Operations Impact)?
- Run ```npm install``` to install all necessary dependencies.

#### What I have learned working on this feature: 
- I understood how redux works and state management

#### Screenshots: [If you made some visual changes to the application please upload screenshots here, or remove this section]
<img width="1438" alt="Screenshot 2019-09-08 at 7 12 00 PM" src="https://user-images.githubusercontent.com/42908597/64527550-21141080-d2fe-11e9-9d74-1ac1b6afaad2.png">
